### PR TITLE
add wrappers for MultilingualPress

### PR DESCRIPTION
**Pull request by vsolovei-smartling**

### CODE REVIEW

File: inc/Smartling/DbAl/MultilingualPressConnector.php [hunk 1]
```diff
@@ -248,18 +253,29 @@ public function unlinkObjects(SubmissionEntity $submission)
...
+        } catch (\Exception $e) {
+            $this->getLogger()->debug('Failed to unlink objects: ' . $e->getMessage());
+            return false;
+        }
     }
 
     private function isMultilingualPluginActive()
     {
-        return class_exists('Mlp_Helpers');
+        $expectedClasses = ['Mlp_Helpers', 'Mlp_Content_Relations', 'Mlp_Site_Relations'];
```
Comment (PavelLoparev) [inline inc/Smartling/DbAl/MultilingualPressConnector.php:272]: Does new code work with an old and new one? Do we need to support both versions of MP?

File: inc/Smartling/DbAl/MultilingualPressConnector.php [hunk 2]
```diff
@@ -286,11 +301,6 @@ public function getBlogLanguageById($blogId)
      */
     public function getBlogNameByLocale($locale)
     {
-        /**
-         * @var \wpdb $wpdb
-         */
-        global $wpdb;
```
Comment (PavelLoparev) [inline inc/Smartling/DbAl/MultilingualPressConnector.php:292]: Maybe for some reason, we needed 2 conenctions? this and `$this->wpdb`?

File: inc/Smartling/DbAl/MultilingualPressConnector.php (hunk 1 above)
Comment (vsolovei-smartling) [reply inc/Smartling/DbAl/MultilingualPressConnector.php:272]: We don't -- the plugin is obsolete for our purposes, we only use the supported of versions if available.

**Review by dimitrystd — APPROVED** [review]

File: inc/Smartling/DbAl/MultilingualPressConnector.php (hunk 2 above)
Comment (vsolovei-smartling) [reply inc/Smartling/DbAl/MultilingualPressConnector.php:292]: No, they're both actually one connection

**Review by PavelLoparev — APPROVED** [review]

**Pull request merged**